proposals/lip-0053.md

```diff
@@ -301,7 +337,9 @@ crossChainUpdateTransactionParams = {
     "required": [
         "sendingChainID",
         "certificate",
-        "activeValidatorsUpdate",
+        "newBLSKeys",
```

Comment:
After fixing all other comments, it might be worth considering merging those 3 params into one, with name `activeValidatorsUpdate` as before and make them part of this object.

Reply:
done.